src/v/CMakeLists.txt

```diff
@@ -167,11 +167,11 @@ else()
   set(GIT_CLEAN_DIRTY "-dev")
 endif()
 
-configure_file(version.cc.in version.cc @ONLY)
+configure_file(version/version.cc.in version/version.cc @ONLY)
 v_cc_library(
   NAME version
-  HDRS version.h
-  SRCS "${CMAKE_CURRENT_BINARY_DIR}/version.cc"
+  HDRS version/version.h
+  SRCS "${CMAKE_CURRENT_BINARY_DIR}/version/version.cc"
```

Comment:
Curious if this was particularly required for storage, given none of the affected files are in storage?

Reply:
Oh, I'm guessing it was required for features table...

Reply:
>Oh, I'm guessing it was required for features table...

yes, this is correct.